views/index.jade

```diff
@@ -50,11 +52,6 @@ block content
           figure.ethereum-explain.scrollme
             div.explain-03.animateme(data-when="span", data-from="0.6", data-to="0.5", data-opacity="0")
 
-          h6.caption(data-when="span", data-from="0.2", data-to="0", data-opacity="0").animateme On traditional server architectures, every application has to set up its own servers that run their own code in isolated silos, making sharing of data hard.
```

Comment:
Hi Alex...This graphic and the description are excellent and successfully convey the message of decentralization as it compares to centralized architectures.   However, the feedback I got on this area was that it was technically heavy and distracted from the main message of welcoming the visitors and encouraging them to explore further.

It deserves a section all to itself.